docker-compose.storage.minio.yml

```diff
@@ -16,7 +16,7 @@ services:
         labels:
             - "traefik.enable=true"
             - "traefik.http.routers.minio.entrypoints=https"
-            - "traefik.http.routers.minio.rule=Host(`${STORAGE_URL:-s3.docker.mender.io}`)||Headers(`X-Forwarded-Host`,`${STORAGE_URL:-s3.docker.mender.io}`)||PathPrefix(`/mender-artifact-storage`)"
+            - "traefik.http.routers.minio.rule=PathPrefix(`/mender-artifact-storage`)||HeadersRegexp(`X-Amz-Date`, `.+`)"
```

Comment:
just wodnering (I am not an expert here) but does logical or here mean that every request with X-Amz-Date will match? does && make sense?

Reply:
yes, requests with `X-Amz-Date` will match. OR is correct, because when you download the artifact with a presigned link, you do not provide the `X-Amz-Date` header.

Reply:
ok then thank you